transmitterIO.js

```diff
@@ -418,9 +422,91 @@ module.exports = (io, extend_sensor_opt) => {
       });
   }
 
-  const sendNewGlucose = (sgv) => {
+  const sendNewGlucose = (sgv, sendSGV) => {
     io.emit('glucose', sgv);
-    xDripAPS.post(sgv);
+
+    if (sendSGV) {
+      xDripAPS.post(sgv);
+    }
+  }
+
+  const stateString = (state) => {
+    switch (state) {
```

Comment:
This is a repeat of the sensor state in `sensor.js`. Can we define it in one place only?

Reply:
Is there a shared module between the browser JavaScript and the server side JavaScript where it can go, or do I need to create a new shares module? Thanks!

Reply:
Rather than looking up the string value for a given state, the Lookout service adds the string value to the glucose structure before sending it to the GUI. The GUI uses the string value from the structure so the code is not duplicated between the Node service and the browser.

Reply:
As you point out, it doesn't seem easy to created a shared module between the browser and server side. Do you think it's cleaner to send the string with the `glucose` message as you've done here, or to duplicate the lookup between the Node service and the browser? I'm happy to merge either.

Reply:
I'm with you preferring not to duplicate code, but I want the string in both places so it displays the decoded value in the log file and the GUI.

I prefer sending the string to duplicating the code. Wish I had thought of that first! 😄